tools/doprofile.py

```diff
@@ -270,10 +270,14 @@ def setXAxisSteps(self):
                     step = step * 365/12
 
                 profile["l"] = []
-                if useNetcdfTime and profile["layer"].source().startswith("NETCDF:"):
+                if useNetcdfTime and (profile["layer"].source().startswith("NETCDF:") or
```

Comment:
Might want to store `profile["layer"].source()` in a variable instead of calling it again and again.